### PR TITLE
Use fracoro data for all new UFS applications 

### LINKED ISSUE #863: FIXfv3 not being set up correctly for atmos-only experiment in the latest workflow
Issue:  
An attempt to run  atmos-only experiment using the latest workflow (develop branch) and the model tag p8c revealed that FIXfv3 is not set up correctly for atmos-only runs if FRAC_GRID=.true..   The new VIIRS veg type datasets and new mask/oro datasets are only made available in the fix_fv3_fracoro directory. 

In the original [exglobal_forecast](https://github.com/yangfanglin/global-workflow/blob/feature/gfsv17/scripts/exglobal_forecast.sh).sh I have the following settings.
```
reso=025
[[ ${CASE} = "C192" ]] && reso=050
[[ ${CASE} = "C96" ]]  && reso=100
frac_grid=${frac_grid:-".true."}
if [ $frac_grid = ".false." ]; then
 FIXfv3=$FIX_DIR/fix_fv3_gmted2010/${CASE}
else
 FIXfv3=$FIX_DIR/fix_fv3_fracoro/${CASE}.mx${reso}_frac
fi
```
I think the latest fix, mask and oro datasets created by Shan/Mike/Helin should work for both fractional and non-fractional grid.   So the default FIXfv3 should be pointing to fix_fv3_fracoro instead of  fix_fv3_gmted2010 for all new UFS applications.   We need to use the new VIIRS veg type,  VIIRS based land-sea-lake mask and the updated oro dataset that was made to be consistent with MOM6, no matter if we are running coupled or atmos-only experiments.   

 fix_fv3_gmted2010 should only be used for running current operational model (GFS.v16)

----------------
Helin checked the model RT settings and found out the RTs are using the correct (new) datasets.

### PR DESCRIPTION
**Pull request by ChunxiZhang-NOAA**
**Description**

The new fracoro data should be used for all new UFS applications no matter if it uses frac_grid or not.

Most problems in Issue[#863](https://github.com/NOAA-EMC/global-workflow/issues/863) have been resolved. However, one problem remains, e.g., the latest fix, mask and oro datasets (fracoro) created by Shan/Mike/Helin should work for both fractional and non-fractional grid.

Note that it also requires the changes in UFS_UTILS. An corresponding PR[#741](https://github.com/ufs-community/UFS_UTILS/pull/741) in UFS_UTILS has been created.

Fixes: #863

Dependency: UFS_UTILS [PR#741](https://github.com/ufs-community/UFS_UTILS/pull/741)

**Type of change**

- [x] New feature (non-breaking change which adds functionality)

**How Has This Been Tested?**

The Forecast-only tests were successful on Orion. The tests are: Atmosphere-only C96 for FRAC_GRID on and off, respectively; Coupled c384 for GRAC_GRID on and off, respectively. However, the test for GLDAS is not done.

### CODE REVIEW

**Comment by WalterKolczynski-NOAA** [comment]
@ChunxiZhang-NOAA is this still a draft? It also needs to be updated with current develop.

**Comment by ChunxiZhang-NOAA** [comment]
@WalterKolczynski-NOAA Yes, it is still a draft. I will check with George tomorrow for the upstream PR.

**Comment by WalterKolczynski-NOAA** [comment]
Today is @ChunxiZhang-NOAA's last day. Do we know who is going to take over this work? It looks like the UFS Utils PR it was waiting for was merged last week.

**Comment by ChunxiZhang-NOAA** [comment]
@WalterKolczynski-NOAA I don't know who will take over this work, but Fanglin will arrange it.  This PR is basically ready, only needs to update the hash for UFS_UTILS to the latest one or 4e673bf. I can do it within the next few hours.

**Comment by ChunxiZhang-NOAA** [comment]
@WalterKolczynski-NOAA  This PR is ready for review.

File: parm/config/config.init [hunk 1]
```diff
@@ -13,9 +13,11 @@ echo "BEGIN: config.init"
 
 export UFS_DIR=${HOMEgfs}/sorc/ufs_utils.fd
 export GDASINIT_DIR=${UFS_DIR}/util/gdas_init
+export GDAS_INIT_DIR=${UFS_DIR}/util/gdas_init
```
Comment (WalterKolczynski-NOAA) [inline parm/config/config.init]: Why are you introducing a second variable that does the exact same thing? I prefer this new form, but the old one would need to be removed and any existing references updated.

**Review by WalterKolczynski-NOAA — CHANGES_REQUESTED** [review]

**Comment by WalterKolczynski-NOAA** [comment]
@yangfanglin who is supposed to be taking this over?

**Comment by yangfanglin** [comment]
@WalterKolczynski-NOAA Can we get this PR committed first since Chunxi said it was ready ?  We can come back to revisit the issue related to GDASINIT_DIR and GDAS_INIT_DIR.   I am afraid if we wait longer this PR will become more difficult to merge.   For now I do not have anyone can take over this PR.

File: parm/config/config.init (hunk 1 above)
Comment (aerorahul) [reply parm/config/config.init]:
```suggestion
```

Comment (aerorahul) [reply parm/config/config.init]: We can commit this suggestion and merge the PR.
If this breaks, then someone will need to be assigned to fix it.

File: parm/config/config.init [hunk 2]
```diff
@@ -16,6 +16,7 @@ export GDASINIT_DIR=${UFS_DIR}/util/gdas_init
 
 export CRES_HIRES=$CASE
 export CRES_ENKF=$CASE_ENKF
+export FRAC_ORO="yes"
```
Comment (aerorahul) [inline parm/config/config.init:19]: What is `FRAC_ORO` and how does it differ from `FRAC_GRID=.true.`?

Comment (aerorahul) [reply parm/config/config.init:19]: Do we need this?  It has been introduced here, but I can't see where it is used.

Comment (aerorahul) [reply parm/config/config.init:19]: I see it is used in `ufs-utils`.  The question still begs asking, how does it differ from `FRAC_GRID`

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Pull request merged**